docs/extend/extend-apiml/api-mediation-onboarding-with-spring-boot-enabler

```diff
@@ -0,0 +1,502 @@
+# Onboarding a Spring Boot based REST API Service
```

Comment:
Save as `.md` file and add `@ComponentScan(value = {
		"org.zowe.apiml.enable"})` mention

Reply:
Added md extension.